cmd/gen-release-notes/templates/minorReleaseNotes.md

```diff
@@ -3,6 +3,11 @@ title: Change Notes
 description: Istio 1.6 release notes.
 weight: 10
 ---
+
```

Comment:
Since this is a draft, does it make sense to change actual numbers above `Istio 1.6` with fictional numbers? `Istio X.Y.Z`?
(Same for the other files)